include/pokemon.h

```diff
@@ -54,7 +55,7 @@ struct PokemonSubstruct3
  /* 0x05 */ u32 spAttackIV:5;
  /* 0x06 */ u32 spDefenseIV:5;
  /* 0x07 */ u32 isEgg:1;
- /* 0x07 */ u32 abilityNum:1;
+ /* 0x07 */ u32 unk7:1;
```

Comment:
If this is now filler, I'd probably name it `filler` instead

Reply:
none of these bytes are actually correctly filled. It looks odd, but I will keep it as it is successfully tested.